.circleci/config.yml

```diff
@@ -120,6 +120,8 @@ jobs:
     working_directory: *working_directory
     environment:
       KAFKA_VERSION: "2.0.1"
+      KAFKA_AUTHORIZER_CLASS_NAME: 'kafka.security.auth.SimpleAclAuthorizer'
```

Comment:
Should these configs be set in the kafka container?

Reply:
I see what you mean but all `environment` are alias from the same yaml anchor and can't be overridden AFAIK. Also depending on the Kafka version `KAFKA_AUTHORIZER_CLASS_NAME` needs to be different so this also can't be added to that one global environment in `kafka-011`.
Please advise.

Reply:
The `Override Values` section of this article https://support.atlassian.com/bitbucket-cloud/docs/yaml-anchors/  shows how to extend anchored values. You should be able to to use that syntax to set the authorizer specifically for each version without needed to redefine the entire environment. Lemmie know if that helps thanks!

Reply:
@rhansen2 Fixed it using that syntax, I have no way to test if it's correct though.

Reply:
You can test by running the circleci jobs locally or by examining the output of the jobs in circleci (I believe you should be able to view the runs for your PR from the details links). It looks like the class name is incorrect for the AclAuthorizer and should be `kafka.security.authorizer.AclAuthorizer`

Reply:
@rhansen2 Indeed good catch on the class name! Will update and see why things fails.

Reply:
@rhansen2 Fixed the test as well as it was blocking the other tests. All green now!